integration/dbt/scripts/dbt-ol

```diff
@@ -53,9 +54,10 @@ def main():
 
     events = processor.parse().events()
 
-    for event in events:
+    for event in tqdm(events, desc="Emitting OpenLineage events:"):
+        time.sleep(0.25)
```

Comment:
why do we need to sleep here?

Reply:
was this so that the recording looks nicer? :)

Reply:
Yes :wink: 
I've removed it minute before you looked.